Library/Formula/kvazaar.rb

```diff
@@ -15,7 +17,9 @@ class Kvazaar < Formula
 
   def install
     system "make", "-C", "src"
-    bin.install "src/kvazaar"
+    bin.install_symlink "src/kvazaar"
```

Comment:
Looks like nothing is getting installed during the stable build except a broken symlink:
```
kvazaar:
 * The installation seems to be empty. Please ensure the prefix
is set correctly and expected files are installed.
The prefix configure/make argument may be case-sensitive.
```